### PR TITLE
GH-68

### PR DESCRIPTION
**Pull request by rnelson0**
Add alias 'list' to 'r10k deploy display' command

This is for GH-68. I locked myself out of the lab, hopeful the CI server will help test the change (still a newb to all this)

### CODE REVIEW

**Comment by rnelson0** [comment]
OK, it passed CI but I can't run the modified r10k where I am to know if this works. I'll check on Monday if someone else hasn't by then.

**Comment by adrienthebo** [comment]
@rnelson0 do you think that `list` is preferable to `display` for showing environments?

**Comment by rnelson0** [comment]
'Puppet module list' is the PL command, so some consistency there wouldn't
hurt.

On Tuesday, June 10, 2014, Adrien Thebo notifications@github.com wrote:

> @rnelson0 https://github.com/rnelson0 do you think that list is
> preferable to display for showing environments?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/adrienthebo/r10k/pull/154#issuecomment-45694016.

## 

Rob Nelson
rnelson0@gmail.com

File: lib/r10k/cli/deploy.rb [hunk 1]
```diff
@@ -118,6 +118,7 @@ module Display
       def self.command
         @cmd ||= Cri::Command.define do
           name  'display'
+		  aliases 'list'
```
Comment (adrienthebo) [inline lib/r10k/cli/deploy.rb]: Sorry for not getting back sooner to this, could you indent this at the same level as `name` ?

**Comment by adrienthebo** [comment]
Fixed up the commit message a little and merged into e628642, this should be released in 1.4.0. Thanks for the contribution!

**Pull request closed without merging**